### ISSUE TITLE
Autogenerated files in the master branch and repo size

### ISSUE THREAD

**Issue by AlexDaniel** [Project-Bonfire/Bonfire#15]
Currently the size of the repo is around **287 MiB**.

For reference, *Ghostbusters.2016.HDRip.XViD-ETRG* on the pirate bay is *706 MiB*, so this repo is about as bad as 47 minutes of the movie.

While it may seem tolerable right now, it is reaching the point at which this will become unacceptable. When I pulled today I had to download a few megabytes of *extra* data, so it is growing noticeably.

For example, [GitHub says](https://help.github.com/articles/what-is-my-disk-quota/):

> We recommend repositories be kept under 1GB each

A more significant issue is that the master branch of this repo is used to store autogenerated files, which I'd say is a misuse and only makes it even more confusing for contributors.

**Comment by siavooshpayandehazad** [comment]
I agree with all the above issues. we should definitely remove everything extra!
